ZConfig/components/logger/loghandler.py

```diff
@@ -25,6 +25,13 @@
 from logging.handlers import HTTPHandler, SMTPHandler
 from logging.handlers import NTEventLogHandler as Win32EventLogHandler
 
+# Export these, they're used in handlers.py
+SysLogHandler = SysLogHandler
+HTTPHandler = HTTPHandler
+SMTPHandler = SMTPHandler
+Win32EventLogHandler = Win32EventLogHandler
```

Comment:
I don't understand what "export these" means, or what that has to do with re-binding the name.

On another note:  couldn't `handlers.py` just import them from `logging.handlers`?